charts/corda/templates/secrets.yaml

```
@@ -55,6 +58,9 @@
     ( dict "username" ( dict "required" true ) "password" ( dict "generate" 12 ) )
   )
 }}
+{{ else if .Values.bootstrap.rbac.enabled }}
+{{- fail "please set credentials for restApiAdmin secret or disable bootstrap RBAC" }}
```

I'd word this slightly differently:
```suggestion
{{- fail "credentials for restApiAdmin must be provided to bootstrap RBAC if DB bootstrap is disabled" }}
```